l10n_it_ddt/models/account.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2014 Abstract (http://www.abstract.it)
+# Copyright Davide Corio <davide.corio@abstract.it>
+# Copyright 2014-2018 Agile Business Group (http://www.agilebg.com)
+# Copyright 2015 Apulia Software s.r.l. (http://www.apuliasoftware.it)
+# Copyright Francesco Apruzzese <f.apruzzese@apuliasoftware.it>
+# Copyright 2018 Simone Rubino - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models, api
+
+
+class AccountInvoice(models.Model):
+
+    _inherit = 'account.invoice'
+
+    carriage_condition_id = fields.Many2one(
+        'stock.picking.carriage_condition', string='Carriage Condition')
+    goods_description_id = fields.Many2one(
+        'stock.picking.goods_description',
+        string='Description of Goods')
+    transportation_reason_id = fields.Many2one(
+        'stock.picking.transportation_reason',
+        string='Reason for Transportation')
+    transportation_method_id = fields.Many2one(
+        'stock.picking.transportation_method',
+        string='Method of Transportation')
+    carrier_id = fields.Many2one(
+        'res.partner', string='Carrier')
+    parcels = fields.Integer('Parcels')
+    weight = fields.Float(string="Weight")
+    gross_weight = fields.Float(string="Gross Weight")
+    volume = fields.Float('Volume')
+    ddt_ids = fields.One2many(
+        'stock.picking.package.preparation', 'invoice_id', string='DDT')
```

Comment:
Il termine inglese che è paragonabile al nostro documento di trasporto è questo.
https://en.wikipedia.org/wiki/Transport_document

Quindi potremmo usare l'acronimo TD al posto dei vari DDT, DdT, ddt ecc., da modificare qui e in tutte le altre parti in cui compare.